openapi/clusters_mgmt/v1/openapi.json

```diff
@@ -15901,6 +15905,15 @@
           "unknown"
         ]
       },
+      "ClusterImageRegistry": {
+        "description": "ClusterImageRegistry represents the configuration for the cluster's internal image registry.\nUnless explicitly set, the image registry is enabled by default.\nThis setting is immutable and cannot be changed after the cluster is created.",
```

Comment:
The second part of this information should be on the attribute and not in the type definition. The second part is:
```
Unless explicitly set, the image registry is enabled by default.\nThis setting is immutable and cannot be changed after the cluster is created
```